examples/csv-io/csv_io.bal

```diff
@@ -46,15 +57,65 @@ function main(string... args) {
         log:printError("An error occurred while processing the records: ",
                         err = e);
     } finally {
-        //Close the text record channel.
-        match csvChannel.close() {
-            error sourceCloseError => {
-                log:printError("Error occured while closing the channel: ",
-                                err = sourceCloseError);
+        // Close the text record channel.
+        closeCSVChannel(csvChannel);
+    }
+
+    // Read the `.CSV` file as a table.
+    csvChannel = io:openCsvFile(srcFileName);
+    table<Salary> salaryTable;
+    try {
+        io:println("Reading  " + srcFileName + " as a table");
+        match csvChannel.getTable(Employee) {
+            table<Employee> employeeTable => {
+                io:println("Creating `salaryTable` using `select` operation");
+                salaryTable = employeeTable.select(getSalary);
```

Comment:
Well we can simply print the data here. I used select so that I can create a table out of the data obtained from the file which could then be written to another file, instead of writing the same table back.

Reply:
IMO having select here gives the impression that we can't directly use the returned table<Employee> employeeTable. We can use that table in any way user want, not just select